cip-drep-metadata/README.md

```diff
@@ -0,0 +1,241 @@
+1. Deal with it as described in CIP-100 (by rendering the raw unformatted document)
+2. Not display the any of the metadata
+
+In either case the tooling SHOULD provide a warning that this has happened and link to the url described in the metadata anchor of the DRep registration.
+
+### Missing
+If the DRep metadata cannot be found at the address specified in the anchor then the tooling SHOULD provide a warning. There is no need to link to the url in the metadata anchor. 
+
+### Witnesses
+DRep Metadata will not follow the CIP-100 specification related to signing the metadata, the [authors property](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0100/README.md#high-level-description) can be left blank without the need for tooling providers to warn their users that the author has not been validated. Instead the author can be derived from the DRep ID associated with the registration. The need for an `authors` field will also be discarded in favour of including usernames inside of the `body` field. For the avoidance of doubt this CIP recommends that the entire authors property be left blank, and that tooling ignore it. 
+
+### Extended Vocabulary
+Like CIP-108, this CIP also extends the potential vocabulary of CIP-100's `body` property. For all of these properties tooling providers MUST be aware that they are responsible for what they display to their users and that these fields could be used for illegal, unsavoury, or innapropriate purposes or language. Therefore such tools SHOULD have a terms of service which they enforce to moderate what they show to their users. The following are a list of properties tooling should expect to encounter:
+
+#### `paymentAddress`
+Dreps may want to recieve tokens for a variety of reasons such as:
+- donations
+- expenses
+- any incentive program
+
+Therefore there will be a `payment_address` field in the metadata where such payments could be sent. This makes such an address public and payments to DReps transparent. 
+
+#### `username`
+- Compulsory
+- A very short freeform text field. Limited to 80 characters.
+- This SHOULD NOT support markdown text styling.
+- Authors MUST use this field for their profile name/ username.
+- Authors SHOULD attempt to make this field unique.
+- Authours SHOULD avoid crass language.
+
+The compulsory nature of this field was controversial because the `username`s cannot be made unique and therefore are open to abuse (by e.g. copycats). However this is not a reason to not include a `username` it a reason for people reviewing governance actions to properly check the whole profile of a DRep before delegating to them. A `username` MUST be included because it is a human readable identifier, it is the property that people reviewing DReps will most likely identify that DRep by even in the presence of copycats.
+
+#### `picture`
+- Optional 
+- A base 64 encoded profile picture
+- Moderation of this image must be handled on the client side to comply with their TOS
```

Comment:
As mentioned elsewhere, is it necessary to specify when and where clients should moderate the contents of the metadata rather than just describing the metadata format itself?

Reply:
I have resolved this now. Thanks for your input.

Reply:
@Crypto2099 @Thomas-Upfield it's disputable whether this is resolved (see https://github.com/cardano-foundation/CIPs/pull/788#discussion_r1548122679) so please let's agree upon a stance regarding moderation at the next CIP meeting (in a few minutes) if possible.